airflow/ui/src/pages/Dag/Code/Code.tsx

```diff
@@ -73,11 +86,35 @@ export const Code = () => {
   return (
     <Box>
       <HStack justifyContent="space-between" mt={2}>
-        {dag?.last_parsed_time !== undefined && (
-          <Heading as="h4" fontSize="14px" size="md">
-            Parsed at: <Time datetime={dag.last_parsed_time} />
-          </Heading>
-        )}
+        <HStack gap={5}>
+          {dag?.last_parsed_time !== undefined && (
+            <Heading as="h4" fontSize="14px" size="md">
+              Parsed at: <Time datetime={dag.last_parsed_time} />
+            </Heading>
+          )}
+
+          {
+            // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
```

Comment:
This is necessary because the ts eslint plugin produces a wrong logic here. This has been fixed, I suspect we need to upgrade the lib, a follow up PR is coming.

Reply:
Oh yeah, I've run into this issue before.